docs/source/api/iohub/device/eyetracker_interface/PupilLabs_Core_Implementation_Notes.rst

```diff
@@ -0,0 +1,465 @@
+`AprilTag Markers <https://docs.pupil-labs.com/core/software/pupil-capture/#markers>`__.
+
+.. image:: https://raw.githubusercontent.com/wiki/pupil-labs/pupil/media/images/pupil-core-render.jpg
+    :width: 700px
+    :align: center
+    :alt: Pupil Core binocular headset with high-speed scene camera
+
+For a detailed overview of wearable *vs* remote eye trackers, check out
+`this Pupil Labs blog post
+<https://pupil-labs.com/blog/news/what-is-eye-tracking/>`_.
+
+Join the Pupil Labs `Discord community <https://pupil-labs.com/chat>`_ to share
+your research and/or questions.
+
+**************************************
+Device, Software, and Connection Setup
+**************************************
+
+Additional Software Requirements
+================================
+
+`Pupil Capture <https://github.com/pupil-labs/pupil/releases/latest>`_
+version v2.0 or newer
+
+Platforms:
+
+-  Windows 10
+-  macOS 10.14 or newer
+-  Ubuntu 16.04 or newer
+
+Supported Models:
+
+-  Pupil Core headset
+
+Setting Up the Eye Tracker
+==========================
+
+1. Follow `Pupil Core's Getting Started
+   guide <https://docs.pupil-labs.com/core/#getting-started>`__ to setup
+   the headset and Capture software
+
+Setting Up PsychoPy
+===================
+
+1. Open ``experiment settings`` in the Builder Window (cog icon in top
+   panel)
+2. Open the ``Eyetracking`` tab
+3. Modify the properties as follows:
+
+   -  Select ``Pupil Labs`` from the ``Eyetracker Device`` drop down menu
+   -  ``Pupil Remote Address`` / ``Port`` - Defines how to connect to
+      Pupil Capture. See Pupil Capture's *Network API* menu to check
+      address and port are correct. PsychoPy will wait the amount of
+      milliseconds declared in ``Pupil Remote Timeout (ms)`` for the
+      connection to be established. An error will be raised if the
+      timeout is reached.
+   -  ``Pupil Capture Recording`` - Enable this option to tell Pupil
+      Capture to record the eye tracker's raw data during the
+      experiment. You can read more about that in `Pupil Capture's
+      official
+      documentation <https://docs.pupil-labs.com/core/software/pupil-capture/#recording>`__.
+      Leave ``Pupil Capture Recording Location`` empty to record to the
+      default
+   -  ``Gaze Confidence Threshold`` - Set the minimum data quality
+      received from Pupil Capture. Ranges from 0.0 (all data) to 1.0
+      (highest possible quality). We recommend using the default value
+      of 0.6.
+   -  ``Pupillometry Only`` - If this mode is selected you will only
+      receive pupillometry data. No further setup is required. If you
+      are interested in gaze data, keep this option disabled and read on
+      below.
+
+.. image:: https://raw.githubusercontent.com/wiki/pupil-labs/pupil/media/images/eye-tracker-properties.png
+    :width: 700px
+    :align: center
+    :alt: Pupil Core eye tracking options, part of PsychoPy experiment settings
+
+Pupillometry + Gaze Mode
+======================
```

Comment:
this needs one more =
Restructured text raises warnings about being 1 character too short

Reply:
```suggestion
========================
```